addons/event/views/event_registration_views.xml

```diff
@@ -184,28 +171,24 @@
                             </a>
                         </div>
                     </t>
-                    <t t-name="kanban-box">
-                        <div t-attf-class="#{!record.active.raw_value ? 'oe_kanban_card_ribbon' : ''} oe_kanban_global_click o_event_registration_kanban container-fluid">
-                            <div t-if="!record.active.raw_value" class="ribbon ribbon-top-right">
-                                <span class="text-bg-danger">Archived</span>
-                            </div>
-                            <div class="row g-0 h-100">
+                    <t t-name="kanban-card" class="ps-0 pe-0">
+                        <div class="container-fluid">
```

Comment:
can't this div be removed?  and the div inside as well actually (by moving the class `row g-0` on the t-name="kanban-card" node)